### ISSUE TITLE
getmail restarting every 15 seconds

### ISSUE THREAD

**Issue by BlackScreen** [christianbur/docker_projects#7]
Hello,
it seems that there is a problem with the getmail implementation, that is not working with the native IPv6 implementation in docker - which is used by mailcow since recently (https://github.com/mailcow/mailcow-dockerized/commit/f0309c56e93464366a003a16ea7cafa36583ac4d).
The effect is, that the container restarts every about 15 seconds. I did not change anything, only updated mailcow.
Any idea how the fix this? Thank you very much for your help!

**Comment by christianbur** [comment]
I'm already using the version too, but without the problem you describe. However, I have no public IPv6 on my internal email server. 
What do the logs say (docker-compose logs)?

**Comment by BlackScreen** [comment]
> I'm already using the version too, but without the problem you describe. However, I have no public IPv6 on my internal email server.
> What do the logs say (docker-compose logs)?

Nothing that helps. It just is starting over. I don't know what happens there.

**Comment by christianbur** [comment]
Then it gets tricky.
What happens if you rename the config file, does the message "MainThread - ERROR: ERROR settings.ini not found!" appear in the log?

**Comment by BlackScreen** [comment]
> What happens if you rename the config file, does the message "MainThread - ERROR: ERROR settings.ini not found!" appear in the log?

Nope, it just exits. Status is "exited", when checking via `docker ps -a`.

**Comment by BlackScreen** [comment]
I also saw a little "bug" in the latest version, to which I updated before writing here:

`2021-08-06 18:24:29,311 - Thread-XYZ - INFO: Login - status: b'mailadress@gmail.com authenticated (Success)'`

I mean the "b'" om the status. Minor, but a little strange.

**Comment by christianbur** [comment]
b'' is a python byte string, so everything ok.

"What happens if you rename the config file, does the message "MainThread - ERROR: ERROR settings.ini not found!" appear in the log?"
-> This is a test to see if the script gives any output at all.

**Comment by BlackScreen** [comment]
I understand. But the output looks ok, as in the connections to the different mailboxes is setup correctly and then after about 15 seconds it restarts - or when I rename the settings.ini it exits and does not restart. No output of that whatsoever.

**Comment by christianbur** [comment]
Please check the mailboxes, there may be an email with special characters that cannot be retrieved by the script.

**Comment by BlackScreen** [comment]
Ok, this is really weird. You are right. I checked all my mailboxes and for some reason - I do not understand why - it works again after I used the mailcow sync option once. No restarts, just working. Thanks for the hint.
I added all my mailboxes, that I have in getmail, to the mailcow sync and sync them with that option once a day. Hopefully this will help to prevent such a problem in the future. :)
